hadoop-ozone/dist/src/main/compose/ozonesecure/test.sh

```diff
@@ -37,7 +37,7 @@ execute_robot_test s3g s3
 
 execute_robot_test scm scmcli
 
-execute_robot_test scm recon
+# execute_robot_test scm recon
```

Comment:
Is it intentional. As I see you updated the robot file as well. Does it work. If not (yet) some comments can be added what is missing. Maybe with a JIRA link.

Reply:
@elek I updated the robot test and docker configs to enable recon as a passive SCM and test the datanodes endpoint. There is still some work needs to be done to support this setup in a secure environment since Datanodes fail to connect to recon and recon fails to connect to SCM with kerberos auth failure. I am planning to handle this in a separate JIRA - https://issues.apache.org/jira/browse/HDDS-3071

Reply:
Thanks a lot. I will add this link to the comment during the commit to make it easier to follow from the code. (Hope you don't mind...)

```
# TODO: https://issues.apache.org/jira/browse/HDDS-3071
# execute_robot_test scm recon
```